DeAceroDLabs/marketplace-frontend
# Feature/dmp 530 Hover effect Button primary & secondary

## What is done

[[DMP-530] Hover effect button](https://deacero.atlassian.net/browse/DMP-530)

## How is done

- Created a `HoverEffect`  class on `Button` component that will be used to create a hover effect on primary and secondary buttons.

## How to test

1. Run the project via `npm start`
2. You should see the Home Screen. There you will see the hover effect on categories buttons.
3. Check all buttons have a correctly usage.
